src/plots/piePlot.ts

```diff
@@ -355,8 +355,15 @@ export module Plots {
           { x: x + measurement.width, y: y + measurement.height }
         ];
 
-        var sliceIndices = corners.map((corner) => this._sliceIndexForPoint(corner));
-        var showLabel = sliceIndices.every((index) => index === datumIndex);
+        var absoluteCenter = { x: this.width() / 2, y: this.height() / 2 };
```

Comment:
This is more like "half width" and "half height" right? I think the code would be easier to read that way:
```Typescript
var showLabel = corners.every((corner) => Math.abs(corner.x) <= this.width() / 2 && Math.abs(corner.y) <= this.height() /2 );
```